epics/cache.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Provides a cache outside of Redux state that can optimistically update state
+ * before an asynchronous API call returns
+ *
+ * @module CacheMiddleware
+ */
+import Rx from 'rxjs';
```

Comment:
Nitpick, in some epics you import `Rx` others you import `Observable` directly. Do we care to be consistent? (see #10 )